app/controllers/concerns/hal_actions.rb

```diff
@@ -94,8 +93,23 @@ def resources_name
     resource_name.pluralize
   end
 
-  def with_params(keys, arel)
-    keys ||= []
+  def decorate_query(res)
+    res = with_sorting(res)
+    res = with_paging(res)
+    res = with_params(res)
+    res
+  end
+
+  def with_sorting(arel)
+    arel.order(id: :desc)
```

Comment:
It's not clear to me that we want to sort by id by default, is there another reason I'm too dense to see why we're breaking this method out to be overridden at all, rather than just returning things sorted from the `resources` methods? If so, does it seem reasonable to have it just return its argument by default?